libmcount/record.c

```diff
@@ -267,6 +267,7 @@ static unsigned save_to_argbuf(void *argbuf, struct list_head *args_spec,
 			if (str) {
 				unsigned i;
 				char *dst = ptr + 2;
+				const unsigned str_limit = 48;
```

Comment:
For this kind of magic number, I prefer using macro constant with capital letters.  Also 48 seems to too small - I don't know what's the good size but 100 looks like reasonable for me.

Reply:
I also see a crash with a long string argument during `uftrace dump`.

Reply:
> For this kind of magic number, I prefer using macro constant with capital letters.

Okay. I will make it as capital letter.

> Also 48 seems to too small - I don't know what's the good size but 100 looks like reasonable for me.
> I also see a crash with a long string argument during uftrace dump.

I made it as 64 before but crashed when I run `uftrace dump`. I'm not sure about it but /usr/bin/gcc does a lot of string comparison that exceeds my expectation.